src/components/timeline/TrackCounterGraph.js

```diff
@@ -44,14 +51,17 @@ type CanvasProps = {|
   +width: CssPixels,
   +height: CssPixels,
   +lineWidth: CssPixels,
+  +graphType: GraphType,
+  +graphStrokeColor: CanvasPattern,
```

Comment:
Is this correct? Shouldn't is just be 'string'?

Reply:
My apologies, the type has been updated to string in the new changes